pkg/runner/runner.go

```diff
@@ -222,7 +222,19 @@ func Run(cfg *Config) error {
 		},
 	)
 
+	pendingTelemetry := map[string]string{
+		"recording_enabled": "false", // Will be written in telemetry table as `promscale_recording_enabled: false`
+		"alerting_enabled":  "false", // `promscale_alerting_enabled: false`
+	}
 	if cfg.RulesCfg.ContainsRules() {
+		pendingTelemetry["recording_enabled"] = "true"
+		pendingTelemetry["alerting_enabled"] = "false"
```

Comment:
Is this line necessary since we do this 4 lines above it?

Reply:
Right, we just need recording_enabled. alerting_enabled is already set to false.